app/controllers/concerns/effective_user.rb

```diff
@@ -0,0 +1,9 @@
+module EffectiveUser
+  def effective_user_id
+    [
+      session[:ial2_recovery_user_id],
+      session[:doc_capture_user_id],
+      current_user&.id,
```

Comment:
This should be UUID, not numeric ID 🤦 

_(We're not always consistent about this, see [recent discussion](https://gsa-tts.slack.com/archives/CNCGEHG1G/p1613576997180500?thread_ts=1613575408.180300&cid=CNCGEHG1G)... granted also, `user_id` is perhaps misleading as the property name in the analytics parameters)_

Reply:
good catch!